modules/react/_examples/stories/examples/compoundComponent/CustomCard.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {
+  createContainer,
+  createElemPropsHook,
+  createModelHook,
+  createSubcomponent,
+  ExtractProps,
+} from '@workday/canvas-kit-react/common';
+import {Card} from '@workday/canvas-kit-react/card';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {Box} from '@workday/canvas-kit-react/layout';
+import {colors} from '@workday/canvas-kit-react/tokens';
+
+export type Theme = 'dark' | 'light';
+interface CustomCardProps extends ExtractProps<typeof Card> {
+  theme?: Theme;
+}
+
+/**
+ * Compound Component Model Hook
+ */
+const useCustomCardModel = createModelHook({
+  defaultConfig: {theme: 'dark' as Theme},
+})(config => {
+  const lightThemeColor = colors.frenchVanilla100;
+  const darkThemeColor = colors.blackPepper300;
+  const inverseColor = color => (color === darkThemeColor ? lightThemeColor : darkThemeColor);
+
+  const [theme, setTheme] = React.useState(config.theme);
+  const [mainColor, setMainColor] = React.useState(
+    config.theme === 'dark' ? darkThemeColor : lightThemeColor
+  );
+  const [contrastColor, setContrastColor] = React.useState(
+    config.theme === 'dark' ? lightThemeColor : darkThemeColor
+  );
+  const state = {theme, mainColor, contrastColor};
+
+  const events = {
+    toggleTheme: () => {
+      setTheme(currentTheme => (currentTheme === 'dark' ? 'light' : 'dark'));
+      setMainColor(inverseColor);
+      setContrastColor(inverseColor);
+    },
+  };
+
+  return {state, events};
+});
+
+/**
+ * Subcomponent Hook
+ */
+const useCustomCardButton = createElemPropsHook(useCustomCardModel)(({state, events}) => ({
+  colors: {
+    default: {
+      background: state.contrastColor,
+      label: state.mainColor,
+    },
+    hover: {},
+    active: {},
+    focus: {},
+    disabled: {},
+  },
+  onClick: () => events.toggleTheme(),
+}));
+
+/**
+ * Subcomponent Component
+ */
+const CustomCardButton = createSubcomponent(PrimaryButton)({
+  displayName: 'CustomCard.Button',
+  modelHook: useCustomCardModel,
+  elemPropsHook: useCustomCardButton,
+})(({children, ...elemProps}) => {
+  return <PrimaryButton {...elemProps}>{children}</PrimaryButton>;
+});
+
+/**
+ * Main Component Hook
+ */
+const useCustomCard = createElemPropsHook(useCustomCardModel)(({state}) => ({
+  backgroundColor: state.mainColor,
+}));
+
+/**
+ * Main Component
+ */
+const CustomCard = createContainer(Card)({
+  displayName: 'CustomCard',
+  subComponents: {
+    Heading: Card.Heading,
+    Body: Card.Body,
+    Button: CustomCardButton,
+  },
+  modelHook: useCustomCardModel,
+  elemPropsHook: useCustomCard,
+})<CustomCardProps>(({theme, children, ...elemProps}) => {
+  return (
+    <Card depth={2} {...elemProps}>
+      {children}
+    </Card>
+  );
+});
+
+/**
+ * Usage
+ */
+export const Basic = () => {
```

Comment:
```suggestion
export const Template = () => {
```

Reply:
Talked about this after standup. We appreciate you following the pattern and using `Basic` here, but it feels less like an example and more of a template.

Reply:
Done!